src/main/java/org/humancellatlas/ingest/submission/SubmissionEnvelope.java

```diff
@@ -127,4 +128,21 @@ public boolean isOpen() {
         List<SubmissionState> states = Arrays.asList(SubmissionState.values());
         return states.indexOf(this.getSubmissionState()) < states.indexOf(SubmissionState.SUBMITTED);
     }
+
+    @JsonIgnore()
+    public boolean isInEditableState() {
+        List<SubmissionState> nonEditableStates = Arrays.asList(
+                SubmissionState.METADATA_VALIDATING,
```

Comment:
I think METADATA_VALIDATING should be excluded, i fear that it will break importing if we include here, the submission can toggle between draft, validating, valid during importing

Reply:
Oh that's worrying though. Because that is the big one that causes issues

Reply:
When you edit a metadata entity, it will go to validating, you shouldnt be able to do anything at this point right? It will cause probems and is what happened in Alexie's demo and the submission was stuck in validating

Reply:
I didn't know that happened to his demo, could it be that it's a different issue? i wonder if you still have that submission? But yeah, let's keep this here and test what happens